week-4-practise/level-2/exercise-1.js

```diff
@@ -9,17 +9,32 @@ We made a start on this for you here:
 */
 
 var bottle = {

```

Comment:
I'm not sure you have fully understood what needs to be done. You are in the right path but some things are missing.

You assign an object called to a variable called `bottle`. This function has an internal variable, called `volume` (with an initial value of 0) and three functions: `fill`, `drink`, and `empty`. The function `fill` is meant to change the value of `volume` to `100`. In your implementation of bottle, the `fill` function never changes the the value of volume. You have the same issue for the function `drink`. It's meant to decrease the value of `volume` by 10, however it currently returns this subtracted value instead of setting is to `volume`